http_parser.c

```diff
@@ -435,6 +435,15 @@ enum http_host_state
   (IS_ALPHANUM(c) || (c) == '.' || (c) == '-' || (c) == '_')
 #endif
 
+#define NEXTHEADERCHAR() \
```

Comment:
hmm... wrapping it around with a while loop while keeping the break is rather tricky, any good tricks for this?